pkgs/top-level/all-packages.nix

```diff
@@ -25799,9 +25799,9 @@ with pkgs;
   ### DEVELOPMENT / GO
 
   # the unversioned attributes should always point to the same go version
-  go = go_1_20;
-  buildGoModule = buildGo120Module;
-  buildGoPackage = buildGo120Package;
+  go = go_1_21;
```

Comment:
We wait usual wait until 1.21.1 before bumping the default version.

Reply:
sure thanks.
I was not aware :)